blackbox/docs/sql/data_types.txt

```diff
@@ -591,28 +591,44 @@ Array types are defined as follows::
 
 .. _data-type-array-literals:
 
-Array Literals
---------------
+Array Constructor
+-----------------
 
-Array literals are given in brackets and may contain zero or more elements.
+Arrays can be written using the array constructor ``ARRAY[]`` or short ``[]``.
+The array constructor is an expression that accepts both literals and expressions
+as its parameters. Parameters may contain of zero or more elements.
 
-Synopsis (literal brackets are given in single quotes)::
+Synopsis (literals are given in single quotes)::
 
-    '[' element [ , ... ] ']'
+    'ARRAY' ? '[' element [ , ... ] ']'
```

Comment:
should be: `[ ARRAY ] '[' expression [ , ... ] ']'`
also remove "(literals are ...)" or keep the old one.
literal strings are always given in single brackets, the old note was a note about the brackets to not mix them with synopsis brackets